wording-guidelines.md

```diff
@@ -87,61 +144,121 @@ example on basic syntactic exposition, but do show surprising outcomes
 and interactions with other rules. For syntax disambiguation rules,
 present an example of a tricky parse.
 
+### Normative requirements and recommendations
+
 Prefer normative requirements if possible; this increases portability.
+
 Sometimes, implementations cannot all provide a feature due to
 environmental limitations or other concerns, but the standard should
-still guide common implementations towards a desirable outcome.  For
-that case, prefix a paragraph with "Recommended practice:" and use
-"should".
+still guide common implementations towards a desirable outcome.
+For that case, prefix a paragraph with "*Recommended practice*:" and use "should".
+
+**Example:** ([[intro.abstract] paragraph 6](https://eel.is/c++draft/intro.abstract#6))
 
-When adding a new conversion, also check overload resolution for a
-corresponding adjustment.
+> *Recommended practice*:
+> An implementation should issue a diagnostic when such an operation is executed.
+
+### Changes to conversions
+
+When adding a new conversion ([[conv]](https://eel.is/c++draft/conv))
+or changing the behavior of an existing one,
+also check overload resolution for a corresponding adjustment.
 
 ## Language
 
-Use grammatical English.  Do not use contractions.  Do not use "you".
+Use grammatical English.
+Do not use contractions.
+Do not use "you".
 
 By default, use present tense and indicative form.
 
-Hyphenate compound adjectives.  Use the Oxford (or serial) comma.
+Hyphenate compound adjectives (e.g. volatile-qualified).
+Use the Oxford (or serial) comma.
+
+The wording in the C++ Standard adheres to
+[ISO/IEC Principles and rules for the structure and drafting of ISO and IEC documents][iso-principles],
```

Comment:
The title is ISO/IEC Directives, Part 2, I think.  And I'm very hesitant to claim we adhere to it; at the very least, our uses of "shall" are in violation,

Reply:
I still think the link is helpful, considering we at least *try* to follow it. And yeah, I've abbreviated the title a bit and taken just the subtitle.